src/components/back-link/_back-link.scss

```diff
@@ -2,7 +2,7 @@
 @import "../../tools/all";
 @import "../../helpers/all";
 
-@include govuk-exports("back-link") {
+@include govuk-exports("govuk/component/back-link") {
```

Comment:
'component' -> 'components'

Reply:
I'd argue for the use of singular

Reply:
It's a bit weird having component, but helper*s*, object*s*, utilitie*s*. But probably not going back and changing now.

Reply:
It's not part of the public api, so not too much trouble